### PR TITLE
Enhancement/#146 creation raw card item

### PR DESCRIPTION
**Pull request by virginiablesa**
Add first card item version #36

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -141,6 +141,18 @@
...
+    <br />
+    <h1>NC_CARD-ITEM</h1>
+    <div class="nc-card-item-container">
+      <nc-card-item 
+        :caption-style="ncCardCaptionStyle"
+        :image="ncCardItemImage"
+        :title="ncCardItemTitle"
+        :title-line-ellipis="ncTitleLineEllipis"
+        :description="ncCardItemDescription"
+        :description-line-ellipis="ncDescritionLineEllipsis"/>
```
Comment (rafagarcia) [inline src/views/Home.vue]:
```suggestion
        :description-line-ellipis="ncDescriptionLineEllipsis"/>
```

File: src/views/Home.vue [hunk 2]
```diff
@@ -204,7 +217,20 @@ export default {
...
+      ncCardItemDescription:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum Excepteur sint occaecat cupidatat non proident, sunt',
+      ncCardItemTitle:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor',
+      ncCardCaptionStyle: {
+        'position': 'absolute',
+        'bottom': '10px',
+        'right': '10px',
+      },
+      ncDescritionLineEllipsis: 2,
```
Comment (rafagarcia) [inline src/views/Home.vue]:
```suggestion
      ncDescriptionLineEllipsis: 2,
```

File: src/views/Home.vue (hunk 1 above)
Comment (virginiablesa) [reply src/views/Home.vue]: Done!

File: src/views/Home.vue (hunk 2 above)
Comment (virginiablesa) [reply src/views/Home.vue]: Done!

**Pull request merged**